youtube_dl/extractor/nhl.py

```diff
@@ -349,3 +354,201 @@ def _real_extract(self, url):
             'thumbnails': thumbnails,
             'formats': formats,
         }
+
+
+class NHLApi:
+    def __init__(self, extractor, auth=None):
+        self.extractor = extractor
+        self.auth = auth
+        if auth:
+            extractor.to_screen("Using cached credentials. Use the --rm-cache-dir option to remove.")
```

Comment:
I agree. Would you suggest adding a specific parameter like `--nhltv-clear-credentials`, or do you have a better suggestion for how to handle this that's more in line with your view of how youtube-dl should work?